wct.conf.json

```diff
@@ -0,0 +1,49 @@
+{
+  "plugins": {
+    "sauce": {
+      "browsers": [
+        {
+          "browserName":  "microsoftedge",
+          "platform":     "Windows 10",
+          "version":      ""
+        },
+        {
+          "browserName":  "internet explorer",
+          "platform":     "Windows 8.1",
+          "version":      "11"
+        },
+        {
+          "browserName":  "safari",
+          "platform":     "OS X 10.12",
+          "version":      "11"
+        },
+        {
+          "browserName":  "safari",
+          "platform":     "OS X 10.11",
+          "version":      "10"
+        },
+        {
+          "browserName":  "safari",
+          "platform":     "OS X 10.11",
+          "version":      "9"
+        },
+        {
+          "browserName":  "chrome",
+          "platform":     "Windows 10",
+          "version":      "beta"
+        },
+        {
+          "browserName":  "chrome",
+          "platform":     "Windows 10",
+          "version":      ""
+        },
+
+        {
+          "browserName":  "firefox",
+          "platform":     "Windows 10",
+          "version":      ""
+        }
+      ]
+    }
+  }
+}
```

Comment:
Is this file part of the PR?

Reply:
The default WCT config (at least when I created the pull request--it may have changed since) didn't include Safari 11 on Sauce yet, which was an issue because it's the only browser on which this issue exists. Therefore I had to add a wct config that forced Safari 11 to be tested. Which in turn leads to the CI failure because the Travis project doesn't have associated Sauce credentials. If WCT has updated their defaults, I can remove this.

Reply:
...but even if WCT has updated their defaults, Travis still needs to be set up to use Sauce, or it will only run tests locally, meaning Firefox and Chrome, which won't test that my change actually fixes the underlying issue.

Reply:
FWIW, patch seems good when I tested with Safari 11 on my local machine.

Reply:
I'm updating the default browser list to run Safari 11:
https://github.com/Polymer/wct-sauce/pull/29